### ISSUE TITLE
System test failures on spanner

### ISSUE THREAD

**Issue by swcloud** [GoogleCloudPlatform/google-cloud-node#2397]
There are 4 failures in spanner

```
  1) Spanner Transactions read only should accept a read timestamp:

      Uncaught AssertionError: 5 === 1
      + expected - actual

      -5
      +1
      
      at system-test/spanner.js:2719:20
      at DestroyableTransform.<anonymous> (src/transaction.js:532:7)
      at endReadableNT (node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:920:12)

  2) Spanner Transactions read only should accept an exact staleness:

      Uncaught AssertionError: 5 === 2
      + expected - actual

      -5
      +2
      
      at system-test/spanner.js:2758:20
      at DestroyableTransform.<anonymous> (src/transaction.js:532:7)
      at endReadableNT (node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:920:12)

  3) Spanner Transactions read only should do an exact read with concurrent updates:

      Uncaught AssertionError: 'v4' === 'v3'
      + expected - actual

      -v4
      +v3
      
      at system-test/spanner.js:2850:24
      at DestroyableTransform.<anonymous> (src/transaction.js:532:7)
      at endReadableNT (node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:920:12)

  4) Spanner Transactions read only should read with staleness & concurrent updates:

      Uncaught AssertionError: 5 === 1
      + expected - actual

      -5
      +1
      
      at system-test/spanner.js:2878:22
      at src/session-pool.js:314:16
```

**Comment by stephenplusplus** [comment]
PR sent: #2406

**Comment by stephenplusplus** [comment]
Looks like we're still seeing a failure:

```
  1) Spanner Transactions read only should accept a read timestamp:

      Uncaught AssertionError [ERR_ASSERTION]: 3 === 2
      + expected - actual

      -3
      +2

      at system-test\spanner.js:2719:20
      at DestroyableTransform.<anonymous> (src\transaction.js:532:7)
      at endReadableNT (node_modules\readable-stream\lib\_stream_readable.js:991:12)
      at _combinedTickCallback (internal/process/next_tick.js:102:11)
      at process._tickCallback (internal/process/next_tick.js:161:9)
```

@callmehiphop could you take a look and see what the correct value for this would be? I explained why I set the assertion to `2` here: https://github.com/GoogleCloudPlatform/google-cloud-node/pull/2406/files#r123763844 -- but now we're getting 3.

**Comment by stephenplusplus** [comment]
@callmehiphop could you see if you can reproduce this locally now? We merged a Spanner PR (#2450) this morning, and [CI has been failing](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-node/1537):

```
  1) Spanner Transactions read only should accept a read timestamp:

      Uncaught AssertionError: 2 === 1
      + expected - actual

      -2
      +1
      
      at system-test/spanner.js:2760:20
      at DestroyableTransform.<anonymous> (src/transaction.js:530:7)
      at endReadableNT (node_modules/readable-stream/lib/_stream_readable.js:992:12)
      at _combinedTickCallback (internal/process/next_tick.js:80:11)
      at process._tickCallback (internal/process/next_tick.js:104:9)

```

**Comment by lukesneeringer** [comment]
Fixed in the re-org.
